tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -104,7 +104,7 @@ public function testListForeignKeysFromExistingDatabase(): void
             new Schema\ForeignKeyConstraint(
                 ['log'],
                 'log',
-                [''],
+                [],
```

Comment:
Since it comes from a newer major release, I'd double check if it can cause any BC break.